src/attrs_strict/_type_validation.py

```diff
@@ -64,29 +66,34 @@ def _validator(instance, attribute, field):
 
 
 def _validate_elements(attribute, value, expected_type):
+    if expected_type is None:
+        return
+
     base_type = _get_base_type(expected_type)
 
-    if base_type is None or base_type == typing.Any:
+    if base_type == typing.Any:
```

Comment:
Why is the none check dropped here ?

Reply:
Not sure to be fair; I assume given the type checker does not complain it's not needed? @keiclone knows probably better